### PR TITLE
Updated test platform with coded ui test related info

### PR DESCRIPTION
**Pull request by acesiddhu**
## Description
Updated test platform with coded ui test related info

### CODE REVIEW

**Review by mayankbansal018 — APPROVED** [review]

File: src/package/nuspec/Microsoft.TestPlatform.nuspec [hunk 1]
```diff
@@ -7,7 +7,7 @@
...
+    <description>This package contains full set of binaries for the Visual Studio Test Platform (vstest). vstest provides a modern, cross platform testing engine that also powers the testing on .NET Core. Supports the following popular test frameworks - MSTest(v1 and v2), xUnit and Nunit with support for extensibility. Visit https://github.com/microsoft/vstest to know more about vstest platform. The package includes support for running Coded UI tests. When running Coded UI tests, you must ensure that the version you choose matches the major version of Visual Studio with which the test binaries were built. For e.g., if your Coded UI test project was built using Visual Studio 2019 (version 16.x), you must use test platform version 16.x. Coded UI test is deprecated (https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes-preview#test-tools) and Visual Studio 2019 (and Test Platform version 16.x) will be the last version with Coded UI test functionality.</description>
```
Comment (singhsarab) [inline src/package/nuspec/Microsoft.TestPlatform.nuspec]: This package contains the full set of binaries for the Visual Studio Test Platform (vstest). It provides a modern, cross platform testing engine that powers the testing on .NET Core as well. It integrates with popular test frameworks like MSTest(v1 and v2), xUnit and Nunit with support for extensibility.

The package supports running Coded UI tests. While running Coded UI tests, you must ensure that the package version matches the major version of Visual Studio used to build the test binaries. For example, if your Coded UI test project was built using Visual Studio 2019 (version 16.x), you must use test platform version 16.x. Coded UI test is deprecated (https://docs.microsoft.com/en-us/visualstudio/releases/2019/release-notes-preview#test-tools) and Visual Studio 2019 (Test Platform version 16.x) will be the last version with Coded UI test functionality.

Visit https://github.com/microsoft/vstest to know more about vstest platform.

Comment (singhsarab) [reply src/package/nuspec/Microsoft.TestPlatform.nuspec]: /cc: @PBoraMsft

Comment (acesiddhu) [reply src/package/nuspec/Microsoft.TestPlatform.nuspec]: done

Comment (singhsarab) [reply src/package/nuspec/Microsoft.TestPlatform.nuspec]: Check the diff,  I have fixed some grammar as well.

Comment (acesiddhu) [reply src/package/nuspec/Microsoft.TestPlatform.nuspec]: done

**Review by singhsarab — APPROVED** [review]

**Pull request merged**